sql/core/src/main/scala/org/apache/spark/sql/execution/WindowExec.scala

```diff
@@ -177,6 +177,8 @@ case class WindowExec(
         case e @ WindowExpression(function, spec) =>
           val frame = spec.frameSpecification.asInstanceOf[SpecifiedWindowFrame]
           function match {
+            case MakeDecimal(AggregateExpression(f, _, _, _), prec, scale) =>
```

Comment:
Oh, my bad. You're right.
I missed recovering that. I'll remove this.
WindowExec.scala is solid and there is no need to be changed.